### PR TITLE
Feature/swagger

### PR DESCRIPTION
**Pull request by vchumachenko**

### CODE REVIEW

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#39`: Build success. Merge approved  

Details: https://jenkins.fokidoki.su:8888/job/docflow/39/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/39/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#42`: Build failed. Merge restricted, please check build log
GitHub pull request #40 of commit c3663888a201cfba63c20c0ffec280740c158377, no merge conflicts.

Details: https://jenkins.fokidoki.su:8888/job/docflow/42/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/42/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#45`: Build failed. Merge restricted, please check build log
GitHub pull request #40 of commit 91587e8563d5df5523a51e4006347948c45e79dc, no merge conflicts.

Details: https://jenkins.fokidoki.su:8888/job/docflow/45/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/45/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#49`: Build failed. Merge restricted, please check build log
GitHub pull request #40 of commit b0f7635c665f429250b0c7b423c17f004ef13ce2, no merge conflicts.

Details: https://jenkins.fokidoki.su:8888/job/docflow/49/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/49/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#50`: Build failed. Merge restricted, please check build log
GitHub pull request #40 of commit 69ebe47bdcb4d0000d424cda24884b19af2103f8, no merge conflicts.

Details: https://jenkins.fokidoki.su:8888/job/docflow/50/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/50/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#51`: Build failed. Merge restricted, please check build log
GitHub pull request #40 of commit 59e7baa81e15c074029196d8b82893c60c9033f3, no merge conflicts.

Details: https://jenkins.fokidoki.su:8888/job/docflow/51/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/51/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#54`: Build success. Merge approved  

Details: https://jenkins.fokidoki.su:8888/job/docflow/54/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/54/

**Comment by FokiDoki** [comment]
Jenkins CI/CD: Build `#55`: Build success. Merge approved  

Details: https://jenkins.fokidoki.su:8888/job/docflow/55/
Blue ocean: https://jenkins.fokidoki.su:8888/blue/organizations/jenkins/docflow/detail/docflow/55/

File: src/main/java/ru/rosatom/documentflow/configuration/SecurityConfiguration.java [hunk 1]
```diff
@@ -63,6 +63,7 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity, JWTAuthFilter
                 .exceptionHandling().authenticationEntryPoint(authenticationEntryPoint)
                 .and()
                 .authorizeHttpRequests()
+                .antMatchers("/**").permitAll()
```
Comment (HukoJlauII) [inline src/main/java/ru/rosatom/documentflow/configuration/SecurityConfiguration.java:66]: поменяй на  .antMatchers("/swagger-ui/**", "/v3/api-docs/**").permitAll()
а то все наши уровни доступа по ролям падают ,если мы разрешаем все маппинги всем

**Review by HukoJlauII — APPROVED** [review]

**Pull request merged**